main/dhcp/APKBUILD

```diff
@@ -28,49 +34,138 @@ source="http://ftp.isc.org/isc/$pkgname/${_realver}/$pkgname-$_realver.tar.gz
 
 builddir="$srcdir/$pkgname-$_realver"
 
-build() {
-	cd "$builddir"
+# secfixes:
+#   4.4.1-r0:
+#     - CVE-2018-5732
+#     - CVE-2018-5733
+
+_configure() {
 	# fix ipv6
 	export CFLAGS="$CFLAGS -D_GNU_SOURCE \
 		-D_PATH_DHCLIENT_CONF='\"/etc/dhcp/dhclient.conf\"'"
+
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
 		--sysconfdir=/etc/dhcp \
-		--with-cli-pid-file=/var/run/dhcp/dhclient.pid \
+		--localstatedir=/var \
+		--with-srv-conf-file=/etc/dhcp/dhcpd.conf \
+		--with-srv-lease-file=/var/lib/dhcp/dhcpd.leases \
+		--with-srv6-lease-file=/var/lib/dhcp/dhcpd6.leases \
 		--with-cli-lease-file=/var/lib/dhcp/dhclient.leases \
+		--with-cli6-lease-file=/var/lib/dhcp/dhclient6.leases \
 		--with-srv-pid-file=/var/run/dhcp/dhcpd.pid \
-		--with-srv-lease-file=/var/lib/dhcp/dhcpd.leases \
+		--with-srv6-pid-file=/var/run/dhcp/dhcpd6.pid \
+		--with-cli-pid-file=/var/run/dhcp/dhclient.pid \
+		--with-cli6-pid-file=/var/run/dhcp/dhclient6.pid \
 		--with-relay-pid-file=/var/run/dhcp/dhcrelay.pid \
-		--enable-dhcpv6 \
+		--with-relay6-pid-file=/var/run/dhcp/dhcrelay6.pid \
+		--with-libbind=no \
+		--enable-dhcpv4o6 \
 		--enable-paranoia \
-		--mandir=/usr/share/man \
-		--infodir=/usr/share/info
+		--enable-log-pid \
+		"$@"
 
+}
+
+build() {
+	cd "$builddir"
+	cp -a "$builddir" "${builddir}-ldap"
+
+	_configure
+	make -j1 -C bind && make
+
+	cd "${builddir}-ldap"
+	_configure \
+		--with-ldap \
+		--with-ldapcrypto \
+		--with-ldap-gssapi
 	make -j1 -C bind && make
 }
 
 check() {
 	cd "$builddir"
 	make check
+
+	cd "${builddir}-ldap"
+	make check
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install
 
 	install -m755 -D "$srcdir"/dhcpd.initd "$pkgdir"/etc/init.d/dhcpd
 	install -m644 -D "$srcdir"/dhcpd.confd "$pkgdir"/etc/conf.d/dhcpd
+	mv "$pkgdir"/etc "$pkgdir/etc.$pkgname"
+	install -m755 -D "$srcdir"/dhcrelay.initd "$pkgdir"/etc/init.d/dhcrelay
+	install -m644 -D "$srcdir"/dhcrelay.confd "$pkgdir"/etc/conf.d/dhcrelay
+	mv "$pkgdir"/etc "$pkgdir"/etc.dhcrelay
+
+	make DESTDIR="$pkgdir" install
+
 	#install -m751 -D "$builddir"/client/scripts/linux "$pkgdir"/sbin/dhclient-script
 	install -m751 -D "$srcdir"/dhclient-script-alpine "$pkgdir"/sbin/dhclient-script
 	install -d "$pkgdir"/var/lib/dhcp
 	install -d "$pkgdir"/var/run/dhcp
 }
 
+dbg() {
```

Comment:
Why are you overriding this? `depends_dbg` is undefined. Is the reason to reset the dependencies?

Reply:
Yes, so that the `depends` from `dhcp` do not apply to the `dhcp-dbg` apk too.

Reply:
Would it perhaps be better to either explicitly make it empty, or define `depends_dbg` (and other similar variables) as empty?

Reply:
I'm happy to include empty variables at the top for any that are currently unset.